### PR TITLE
Update ports usage doc

### PR DESCRIPTION
**Pull request by bianpengyuan**
Fix https://github.com/istio/istio/issues/26935

### CODE REVIEW

File: content/en/docs/ops/deployment/requirements/index.md [hunk 1]
```diff
@@ -60,22 +60,27 @@ requirements:
...
-| 15021 | HTTP | Envoy | Health checks |
-| 15090 | HTTP | Envoy | Envoy Prometheus telemetry |
-| 15010 | GRPC | Istiod | XDS and CA services (plaintext) |
-| 15012 | GRPC | Istiod | XDS and CA services (TLS) |
-| 8080 | HTTP | Istiod | Debug interface |
-| 443 | HTTPS | Istiod | Webhooks |
-| 15014 | HTTP | Istiod | Control plane monitoring |
+| 15000 | TCP | Envoy admin port (commands/diagnostics) | Yes |
+| 15001 | TCP | Envoy Outbound | Yes |
+| 15006 | TCP | Envoy Inbound | Yes |
```
Comment (howardjohn) [inline content/en/docs/ops/deployment/requirements/index.md:69]: these two are not localhost

**Review by howardjohn — APPROVED** [review]

File: content/en/docs/ops/deployment/requirements/index.md (hunk 1 above)
Comment (bianpengyuan) [reply content/en/docs/ops/deployment/requirements/index.md]: Done.

File: content/en/docs/ops/deployment/requirements/index.md [hunk 2]
```diff
@@ -60,22 +60,27 @@ requirements:
 
 ## Ports used by Istio
 
-The following ports and protocols are used by Istio.
+The following ports and protocols are used by Istio sidecar proxy.
 
-| Port | Protocol | Used by | Description |
+| Port | Protocol | Description | Localhost Only |
```
Comment (ericvn) [inline content/en/docs/ops/deployment/requirements/index.md:65]:
```suggestion
| Port | Protocol | Description | Local host Only |
```

File: content/en/docs/ops/deployment/requirements/index.md [hunk 3]
```diff
@@ -60,22 +60,27 @@ requirements:
...
+| 15001 | TCP | Envoy Outbound | No |
+| 15006 | TCP | Envoy Inbound | No |
+| 15008 | TCP | Envoy Tunnel port (Inbound) | No |
+| 15020 | HTTP | Merged Prometheus telemetry from Istio agent, Envoy, and application | No |
+| 15021 | HTTP | Health checks | No |
+| 15090 | HTTP | Envoy Prometheus telemetry | No |
+
+The following ports and protocols are used by Istio control plane.
+
+| Port | Protocol | Description | Localhost Only |
```
Comment (ericvn) [inline content/en/docs/ops/deployment/requirements/index.md:77]:
```suggestion
| Port | Protocol | Description | Local host Only |
```

File: content/en/docs/ops/deployment/requirements/index.md [hunk 4]
```diff
@@ -60,22 +60,27 @@ requirements:
...
+|----|----|----|
+| 15010 | GRPC | XDS and CA services (Plaintext) | No |
+| 15012 | GRPC | XDS and CA services (TLS, recommanded for production usage) | No |
```
Comment (ericvn) [inline content/en/docs/ops/deployment/requirements/index.md:80]:
```suggestion
| 15012 | GRPC | XDS and CA services (TLS, recommended for production usage) | No |
```

**Review by ericvn — COMMENTED** [review]
Several spelling errors causing lint to fail

File: content/en/docs/ops/deployment/requirements/index.md [hunk 5]
```diff
@@ -60,22 +60,27 @@ requirements:
 
 ## Ports used by Istio
 
-The following ports and protocols are used by Istio.
+The following ports and protocols are used by Istio sidecar proxy.
```
Comment (frankbu) [inline content/en/docs/ops/deployment/requirements/index.md:63]:
```suggestion
The following ports and protocols are used by the Istio sidecar proxy (Envoy).
```

File: content/en/docs/ops/deployment/requirements/index.md [hunk 6]
```diff
@@ -60,22 +60,27 @@ requirements:
...
 
-| Port | Protocol | Used by | Description |
+| Port | Protocol | Description | Local host only |
```
Comment (frankbu) [inline content/en/docs/ops/deployment/requirements/index.md:65]: What does "local host only" (true or false) mean? Why should readers care?

File: content/en/docs/ops/deployment/requirements/index.md [hunk 7]
```diff
@@ -60,22 +60,27 @@ requirements:
...
-| 15014 | HTTP | Istiod | Control plane monitoring |
+| 15000 | TCP | Envoy admin port (commands/diagnostics) | Yes |
+| 15001 | TCP | Envoy Outbound | No |
+| 15006 | TCP | Envoy Inbound | No |
+| 15008 | TCP | Envoy Tunnel port (Inbound) | No |
+| 15020 | HTTP | Merged Prometheus telemetry from Istio agent, Envoy, and application | No |
+| 15021 | HTTP | Health checks | No |
+| 15090 | HTTP | Envoy Prometheus telemetry | No |
+
+The following ports and protocols are used by Istio control plane.
```
Comment (frankbu) [inline content/en/docs/ops/deployment/requirements/index.md:75]:
```suggestion
The following ports and protocols are used by the Istio control plane (istiod).
```

File: content/en/docs/ops/deployment/requirements/index.md [hunk 8]
```diff
@@ -60,22 +60,27 @@ requirements:
...
+
+| Port | Protocol | Description | Local host only |
+|----|----|----|----|
+| 15010 | GRPC | XDS and CA services (Plaintext) | No |
+| 15012 | GRPC | XDS and CA services (TLS, recommended for production usage) | No |
```
Comment (frankbu) [inline content/en/docs/ops/deployment/requirements/index.md:80]:
```suggestion
| 15012 | GRPC | XDS and CA services (TLS, recommended for production use) | No |
```

File: content/en/docs/ops/deployment/requirements/index.md [hunk 9]
```diff
@@ -60,22 +60,27 @@ requirements:
...
-| 15020 | HTTP | Envoy | Istio agent Prometheus telemetry |
-| 15021 | HTTP | Envoy | Health checks |
-| 15090 | HTTP | Envoy | Envoy Prometheus telemetry |
-| 15010 | GRPC | Istiod | XDS and CA services (plaintext) |
-| 15012 | GRPC | Istiod | XDS and CA services (TLS) |
-| 8080 | HTTP | Istiod | Debug interface |
-| 443 | HTTPS | Istiod | Webhooks |
-| 15014 | HTTP | Istiod | Control plane monitoring |
+| 15000 | TCP | Envoy admin port (commands/diagnostics) | Yes |
+| 15001 | TCP | Envoy Outbound | No |
```
Comment (frankbu) [inline content/en/docs/ops/deployment/requirements/index.md:68]:
```suggestion
| 15001 | TCP | Envoy outbound | No |
```

File: content/en/docs/ops/deployment/requirements/index.md [hunk 10]
```diff
@@ -60,22 +60,27 @@ requirements:
...
+| 15006 | TCP | Envoy Inbound | No |
```
Comment (frankbu) [inline content/en/docs/ops/deployment/requirements/index.md:69]:
```suggestion
| 15006 | TCP | Envoy inbound | No |
```

File: content/en/docs/ops/deployment/requirements/index.md [hunk 11]
```diff
@@ -60,22 +60,27 @@ requirements:
...
+| 15008 | TCP | Envoy Tunnel port (Inbound) | No |
```
Comment (frankbu) [inline content/en/docs/ops/deployment/requirements/index.md:70]:
```suggestion
| 15008 | TCP | Envoy tunnel port (inbound) | No |
```

File: content/en/docs/ops/deployment/requirements/index.md (hunk 6 above)
Comment (bianpengyuan) [reply content/en/docs/ops/deployment/requirements/index.md:65]: I think this is useful. It tells user which port can be visited from the outside. Although I see your point that local host only is kind of confusing here.. @costinm @howardjohn any suggestion on the naming?

Comment (frankbu) [reply content/en/docs/ops/deployment/requirements/index.md:65]: From outside of what? Outside of the pod? If so, how about:
```suggestion
| Port | Protocol | Description | Pod-internal only |
```

File: content/en/docs/ops/deployment/requirements/index.md [hunk 12]
```diff
@@ -60,22 +60,27 @@ requirements:
...
+| 15021 | HTTP | Health checks | No |
+| 15090 | HTTP | Envoy Prometheus telemetry | No |
+
+The following ports and protocols are used by the Istio control plane (istiod).
+
+| Port | Protocol | Description | Local host only |
+|----|----|----|----|
+| 15010 | GRPC | XDS and CA services (Plaintext) | No |
+| 15012 | GRPC | XDS and CA services (TLS, recommended for production use) | No |
+| 8080 | HTTP | Debug interface (Deprecated) | No |
```
Comment (frankbu) [inline content/en/docs/ops/deployment/requirements/index.md:81]:
```suggestion
| 8080 | HTTP | Debug interface (deprecated) | No |
```

File: content/en/docs/ops/deployment/requirements/index.md (hunk 6 above)
Comment (howardjohn) [reply content/en/docs/ops/deployment/requirements/index.md:65]: "local host only", "pod internal", or "Exposed externally" (and inverting) all sound equally good to me

Comment (frankbu) [reply content/en/docs/ops/deployment/requirements/index.md:65]: In that case, I recommend going with "Pod-internal". It's the least ambiguous IMO.

**Review by frankbu — APPROVED** [review]

**Pull request merged**